.github/workflows/ci-test.yaml

```
@@ -0,0 +1,27 @@
+name: ci-test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: src
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.x"
+
+      - name: Test
```

```suggestion
      - name: Verify metadata files
```

Or maybe "Metadata Test", something not too general.